kyaml/fn/framework/framework.go

```diff
@@ -170,3 +321,120 @@ func (rl *ResourceList) Filter(api kio.Filter) error {
 	}
 	return nil
 }
+
+// SortItems sorts the ResourceList.items by apiVersion, kind, namespace and name.
+func (rl *ResourceList) SortItems() {
+	sort.Sort(sortRNodeObjects(rl.Items))
```

Comment:
kio's `filters` package already includes `FormatFilter`, which also sorts fields. Should we wrap that instead?